docs/conf.py

```diff
@@ -0,0 +1,323 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2015 CERN.
```

Comment:
Please follow [invenio-query-parser](https://github.com/inveniosoftware/invenio-query-parser) style that was one of the first separately published packages.  If cookiecutter uses only generic term Invenio, then we may indeed want to change that... 

(The thinking behind using concrete package name in the copyright header blurb was that packages may be independent, may have different contributing institutes (hence copyright statements), different licenses, etc.)